infra/shared/appConfiguration.bicep

```diff
@@ -40,29 +40,37 @@ resource variantFeatureFlagGreeting 'Microsoft.AppConfiguration/configurationSto
       "enabled": true,
       "variants": [
         {
-          "name": "On",
-          "configuration_value": true
+          "name": "None"
```

Comment:
I know this is a string, but can we avoid None as it's a null in python and it looks confusing like that and we will have it for all languages. How about `No Message`.

Reply:
What about `Default` for this variant and the percentage variant below as well?

Reply:
`Default` sounds good to me.